pir/cpp/client.h

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef PIR_CLIENT_H_
+#define PIR_CLIENT_H_
+
+#include <string>
+
+#include "context.h"
+#include "util/statusor.h"
+
+namespace pir {
+
+using ::private_join_and_compute::StatusOr;
+
+class PIRClient {
+ public:
+  /**
+   * Creates and returns a new client instance.
+   * @returns TODO
```

Comment:
You've left a lot of these as TODO. I'd suggest either writing what they are now, or removing them.

Reply:
Yes, true. The left TODOs are related to documented the errors.
If the current interface is fine, I will review the error handling, now there's a lot of ValueOrDie.

Reply:
error handling updated